<?xml version="1.0"?>

<!-- map.launch -->
<launch>
    <!-- Waypoints -->
    <include file="$(find igvc)/launch/waypoint_source.launch" />

    <!-- Planner -->
    <node name="path_planner" pkg="igvc" type="pathplanner" output="screen" >
        <param name="goal_threshold" type="double" value="0.8"/>
        <param name="c_space" type="double" value="0.5"/>
        <param name="probability_threshold" type="double" value="0.7"/>
        <param name="point_turns_enabled" type="bool" value="false"/>
        <param name="reverse_enabled" type="bool" value="false"/>
        <param name="max_jump_size" type="double" value="10.0"/>
        <param name="theta_filter" type="double" value="1.5"/>
        <param name="max_theta_change" type="double" value="0.8"/>
        <param name="theta_change_window" type="double" value="3.0"/>
    </node>

    <!-- Follower -->
    <node name="path_follower" pkg="igvc" type="path_follower" output="screen" >
        <param name="maximum_vel" type="double" value="6.0"/>
        <param name="target_v" type="double" value="0.6"/>
        <param name="axle_length" type="double" value="0.52"/>
        <param name="lookahead_dist" type="double" value="1.8"/>
        <param name="k1" type="double" value="0.5"/>
        <param name="k2" type="double" value="1.0"/>
    </node>

</launch>
